omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/RestTestConstants2_2.java

```diff
@@ -16,4 +16,8 @@
 	public final static String PROGRAM_ATTRIBUTE_TYPE_UUID = "d7477c21-bfc3-4922-9591-e89d8b9c8efb";
 	
 	public final static String PATIENT_PROGRAM_UUID = "9119b9f8-af3d-4ad8-9e2e-2317c3de91c6";
+	
+	public final static String CONDITION_UUID = "c804ee60-ecbc-4d70-abda-1e4f6f64e5b5";
+	
+	public static final String CONDITION_TEST_DATA_XML = "conditionTestDataSet.xml";
```

Comment:
This file will not be found. It needs the full path as here https://github.com/openmrs/openmrs-core/pull/2590/commits/d4981cd723b8dd55064c1b58ae6b0bcc4a19665f#diff-0d16b7abc7a5851414804ac8d1b34bb9R31